### PR TITLE
sb-all-storylingo-games integrate

### PR DESCRIPTION
**Pull request by sagarkoshti1990**

### CODE REVIEW

File: src/app/views/pages/home/Tamil.html [hunk 1]
```diff
@@ -1,357 +1,483 @@
 <!DOCTYPE html>
```
Comment (gouravmore) [inline src/app/views/pages/home/Tamil.html]: @sagarkoshti1990 is this change required?

Comment (sagarkoshti1990) [reply src/app/views/pages/home/Tamil.html]: it was just a format change. i have reverted it.
